paperqa/clients/utils.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Iterable
+from functools import reduce
+from http import HTTPStatus
+from typing import Any
+
+import aiohttp
+import httpx
+from tenacity import (
+    before_sleep_log,
+    retry,
+    retry_if_exception,
+    stop_after_attempt,
+    wait_incrementing,
+)
+
+logger = logging.getLogger(__name__)
+
+TITLE_SET_SIMILARITY_THRESHOLD = 0.75
```

Comment:
Can you add a comment explaining units here?

Reply:
i'm just cutting it for hard coding into the defaults in the respective methods, it's also in the model itself